cmd/preview/create.go

```diff
@@ -0,0 +1,112 @@
+package preview
+
+import (
+	"context"
+	"fmt"
+	preview "github.com/armory-io/preview-service/pkg/client"
+	"github.com/armory/armory-cli/pkg/cmdUtils"
+	"github.com/armory/armory-cli/pkg/config"
+	"github.com/mitchellh/go-homedir"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+	"os"
+	"path"
+	"time"
+)
+
+const (
+	createShort = "Create a network preview"
+	createLong  = "Create a network preview"
+
+	clusterPreviewType = "cluster"
+)
+
+type (
+	createPreviewOptions struct {
+		Type     string
+		Duration string
+		Agent    string
+	}
+)
+
+func NewCmdCreate(configuration *config.Configuration) *cobra.Command {
+	logger := zap.S()
+	options := &createPreviewOptions{}
+
+	cmd := &cobra.Command{
+		Use:     "create",
+		Aliases: []string{},
+		Short:   createShort,
+		Long:    createLong,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			assertEqual(logger, clusterPreviewType, options.Type, fmt.Sprintf("preview type must be %q, got %q", clusterPreviewType, options.Type))
+
+			client := preview.NewClient(func(ctx context.Context) (string, error) {
+				return configuration.GetAuthToken(), nil
+			}, configuration.GetArmoryCloudAddr().String())
+
+			duration, err := time.ParseDuration(options.Duration)
+			assertNil(logger, err, "Provided duration is not a valid Go duration string")
+
+			p, err := client.CreateClusterPreview(cmd.Context(), preview.ClusterPreviewParameters{
+				AgentIdentifier: options.Agent,
+				Duration:        duration,
+			})
+			assertNil(logger, err, "Could not create cluster preview")
+
+			home, err := homedir.Dir()
+			assertNil(logger, err, "Could not determine $HOME directory")
+			assertNil(
+				logger,
+				client.UpdateKubeconfigWithClusterPreview(*p, path.Join(home, ".kube", "config")),
+				"Could not update ~/.kube/config with cluster preview",
+			)
+			return nil
```

Comment:
```suggestion
			cmd.Println("Your Kubernetes config has been updated with the preview context.")
			return nil
```

Reply:
Much to your comment on my last PR for the CLI about not being sure what to do next or what happened, etc., this command finishes with no output. It isn't necessarily obvious that we've just updated their kube config file.  These may not be the exact right words to use :)